Header Header
float64 Latitude
float64 Longitude
float64 Altitude
float64 HDOP
float64 UTM_easting
float64 UTM_northing
time    UTC
float64 Zone
string Letter
